these confuse the test driver for some reason (they're fine when being run individually..?)

for example, the test output driver is:
    # test_get_max_cpuid_from_mask: Testing smp helpers
    PASS: test_get_max_cpuid_from_mask 1 - get_max_cpuid_from_mask '', expected: '-1', result: '-1'
    ERROR: test_get_max_cpuid_from_mask 30 - get_max_cpuid_from_mask '1 # OUT-OF-ORDER (expecting 2)
    ERROR: test_get_max_cpuid_from_mask - too few tests run (expected 36, got 2)
    tap-driver.sh: internal error getting exit status
    tap-driver.sh: fatal: I/O or internal error
but the test executable itself prints the results 1 to 30 in order

diff -ruN a/tests/unit/Makefile.am b/tests/unit/Makefile.am
--- a/tests/unit/Makefile.am	2025-04-14 20:11:29.000000000 +0200
+++ b/tests/unit/Makefile.am	2025-07-05 10:46:34.159176832 +0200
@@ -47,26 +47,8 @@
 	test_arch_cxx \
 	test_get_cpu_mask_from_sysfs \
 	test_get_cpu_mask_from_sysfs_cxx \
-	test_get_max_cpuid_from_mask \
-	test_get_max_cpuid_from_mask_cxx \
 	test_get_max_cpuid_from_sysfs \
-	test_get_max_cpuid_from_sysfs_cxx \
-	test_get_possible_cpus_array_len \
-	test_get_possible_cpus_array_len_cxx \
-	test_uatomic \
-	test_uatomic_cxx \
-	test_urcu_multiflavor \
-	test_urcu_multiflavor_cxx \
-	test_urcu_multiflavor_dynlink \
-	test_urcu_multiflavor_dynlink_cxx \
-	test_urcu_multiflavor_single_unit \
-	test_urcu_multiflavor_single_unit_cxx \
-	test_urcu_multiflavor_single_unit_dynlink \
-	test_urcu_multiflavor_single_unit_dynlink_cxx \
-	test_build \
-	test_build_cxx \
-	test_build_dynlink \
-	test_build_dynlink_cxx
+	test_get_max_cpuid_from_sysfs_cxx
 
 noinst_HEADERS = test_urcu_multiflavor.h
 
